src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
This check was added/needed to match on migrations behavior where reserved words are being converted to lower case.